static/src/stylesheets/module/content-garnett/_content.scss

```diff
@@ -1094,8 +1094,27 @@
     }
     .content__standfirst,
     .content__dateline,
-    .caption--main {
-        color: $neutral-2-contrasted;
+    .caption--main,
+    .caption {
+        color: $garnett-neutral-7;
+    }
+    .inline-icon {
+        fill: $garnett-neutral-7;
+    }
+    a {
+        color: $garnett-neutral-1;
+    }
+    .u-underline {
+        text-decoration: none !important;
```

Comment:
Does this need to be `!important`? What is it overriding?

Reply:
I have copied the underline rule from the body text in the gu content.

Reply:
[Here?](https://github.com/guardian/frontend/blob/master/static/src/stylesheets/base/_helpers.scss#L216-L229)

Maybe you only need to override the border colours?

Reply:
Good point. I will take a look it now.